From b91f178edf3ba6969120a8ad2f5a7a4d330e2b64 Mon Sep 17 00:00:00 2001
From: Alexandre Courbot <acourbot@nvidia.com>
Date: Tue, 20 Dec 2016 15:20:20 +0900
Subject: [PATCH 4/4] remove dependency on libdrm_tegra

Functions provided by this library are not required as we perform ioctls
directly.

Forward port to mesa 17.0.2
Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 configure.ac                             | 2 --
 src/gallium/winsys/tegra/drm/Makefile.am | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index f7f94ee..4ce5d37 100644
--- a/configure.ac
+++ b/configure.ac
@@ -74,7 +74,6 @@ LIBDRM_INTEL_REQUIRED=2.4.61
 LIBDRM_NVVIEUX_REQUIRED=2.4.66
 LIBDRM_NOUVEAU_REQUIRED=2.4.66
 LIBDRM_FREEDRENO_REQUIRED=2.4.74
-LIBDRM_TEGRA_REQUIRED=2.4.68
 LIBDRM_VC4_REQUIRED=2.4.69
 LIBDRM_ETNAVIV_REQUIRED=2.4.74
 DRI2PROTO_REQUIRED=2.6
@@ -2393,7 +2392,6 @@ if test -n "$with_gallium_drivers"; then
             require_basic_egl "virgl"
             ;;
         xtegra)
-            PKG_CHECK_MODULES([TEGRA], [libdrm_tegra >= $LIBDRM_TEGRA_REQUIRED])
             HAVE_GALLIUM_TEGRA=yes
             gallium_require_drm "tegra"
             gallium_require_drm_loader
diff --git a/src/gallium/winsys/tegra/drm/Makefile.am b/src/gallium/winsys/tegra/drm/Makefile.am
index 39068fe..eada86a 100644
--- a/src/gallium/winsys/tegra/drm/Makefile.am
+++ b/src/gallium/winsys/tegra/drm/Makefile.am
@@ -26,7 +26,7 @@ include $(top_srcdir)/src/gallium/Automake.inc
 AM_CFLAGS = \
 	-I$(top_srcdir)/src/gallium/drivers \
 	$(GALLIUM_WINSYS_CFLAGS) \
-	$(TEGRA_CFLAGS)
+	$(LIBDRM_CFLAGS)
 
 noinst_LTLIBRARIES = libtegradrm.la
 
-- 
2.9.3

